archivers/wimlib/Portfile

```diff
@@ -29,21 +29,32 @@ master_sites        ${homepage}/downloads/
 depends_build       port:pkgconfig
 
 depends_lib         port:libxml2 \
-                    port:ntfs-3g \
-                    path:lib/libssl.dylib:openssl \
-                    port:osxfuse
+                    path:lib/libssl.dylib:openssl
 
 depends_run         port:cabextract \
                     port:cdrtools \
                     port:mtools
 
-# ntfs-3g is not universal
-universal_variant   no
-
 configure.args      --disable-silent-rules \
+                    --with-libcrypto \
                     --without-fuse \
-                    --with-ntgs-3g \
-                    --with-libcrypto
+                    --without-ntfs-3g
+
+variant fuse description {Mount WIM images with FUSE} {
+  depends_lib-append      port:osxfuse
+  configure.args-replace  --without-fuse --with-fuse
+
+  # osxfuse is not universal
+  universal_variant       no
```

Comment:
You can't make the existence of a variant (e.g. the universal variant) contingent on whether another variant is selected. You can however mark these new variants as conflicting with the universal variant. But in fact we have to keep the universal variant disabled for the entire port because another one of its dependencies, cdrtools, does not have a universal variant.

Reply:
Actually since cdrtools is a runtime dependency, that means we're only using its programs, not its libraries, so we can skip the architecture checking and offer a universal variant after all.